<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PsyLab BloodOxy Plotter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="canovas" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>870</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_F">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>200</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Select local file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AZ">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>594</y>
      <width>86</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyze</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>600</y>
      <width>86</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>MX/OX Graph</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="canovas_3" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>821</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelF">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>240</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>No file sellected</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_L">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>460</y>
      <width>44</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>30</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>494</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Min HR</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_U">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>460</y>
      <width>44</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>90</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>494</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Max HR</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Remote">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>34</y>
      <width>102</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Re&amp;mote</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Local">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>34</y>
      <width>102</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Local</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>74</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>280</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>MV merge</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>100</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DL">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>160</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>2</number>
    </property>
    <property name="text">
     <string>Download file</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit_U_2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>565</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>8</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2000</year>
      <month>1</month>
      <day>2</day>
     </datetime>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2000</year>
      <month>1</month>
      <day>2</day>
     </datetime>
    </property>
    <property name="minimumDate">
     <date>
      <year>2000</year>
      <month>1</month>
      <day>2</day>
     </date>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::HourSection</enum>
    </property>
    <property name="displayFormat">
     <string>hh:mm</string>
    </property>
    <property name="timeSpec">
     <enum>Qt::UTC</enum>
    </property>
    <property name="time">
     <time>
      <hour>8</hour>
      <minute>0</minute>
      <second>0</second>
     </time>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit_L_2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>525</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>20</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2000</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="maximumDate">
     <date>
      <year>2000</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
    <property name="minimumDate">
     <date>
      <year>2000</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::HourSection</enum>
    </property>
    <property name="displayFormat">
     <string>hh:mm</string>
    </property>
    <property name="timeSpec">
     <enum>Qt::UTC</enum>
    </property>
    <property name="time">
     <time>
      <hour>20</hour>
      <minute>0</minute>
      <second>0</second>
     </time>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_O">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>600</y>
      <width>86</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>HR Graph</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>320</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>HR merge</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>360</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>OX merge</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>530</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Time start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>570</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Time end</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_L">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>525</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Day1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Day2</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_U">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>565</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Day2</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>Day1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Day2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>430</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>OD merge</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_ODI_Dots">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>390</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OD events</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_ODI_List">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>390</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OD events list</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Noise Gap Threshold</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_Filter">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>600</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_m">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>275</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>300</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_H">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>315</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_O">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>355</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_OD">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>425</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>570</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Oxegenation string</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>600</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LB</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>600</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_clipperLB">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>600</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>50</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>74</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_clipperStep">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>600</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clipper">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>600</y>
      <width>86</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate string</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_s">
    <property name="geometry">
     <rect>
      <x>975</x>
      <y>75</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>00XX00YY</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_d">
    <property name="geometry">
     <rect>
      <x>975</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>yyyy-MM-dd</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHRV">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <widget class="QMenu" name="menuServer">
     <property name="title">
      <string>Server</string>
     </property>
     <addaction name="action120_126_32_35"/>
     <addaction name="actionxenon_ym_edu_tw"/>
    </widget>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionEnglish_en_US"/>
     <addaction name="action_zh_tw"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="actionReset_plot"/>
    <addaction name="menuServer"/>
    <addaction name="menuLanguage"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuHRV"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionReset_plot">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Reset plot</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action120_126_32_35">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>xenon2_ym_edu_tw</string>
   </property>
  </action>
  <action name="actionxenon_ym_edu_tw">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>xenon_ym_edu_tw</string>
   </property>
  </action>
  <action name="action_white_ox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>移除白噪音（血氧）</string>
   </property>
  </action>
  <action name="actionEnglish_en_US">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English (en-US)</string>
   </property>
  </action>
  <action name="action_zh_tw">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>中文 (台灣)(zh_tw)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
